recipes/netcdf/all/conanfile.py

```diff
@@ -32,100 +34,92 @@ class NetcdfConan(ConanFile):
         "netcdf4": True,
         "with_hdf5": True,
         "cdf5": True,
-        "dap": True,
+        "dap": False,
```

Comment:
Why was this changed? https://github.com/Unidata/netcdf-c/blob/0f29b454d282ea2f894648d533fa1ba045ef40c4/CMakeLists.txt#L513

Reply:
Note in one of my previous commits: `Set default for 'dap' option to False, to avoid openssl dependency`
I can reenable it if openssl can build on CCI now...

Reply:
If openssl doesn't  build that's a bigger problem  😱 

I think it should be enabled, I just saw the new version was merged in too

Reply:
This is done now